driver-reactive-streams/src/main/com/mongodb/reactivestreams/client/internal/gridfs/GridFSBucketImpl.java

```diff
@@ -237,8 +261,11 @@ public GridFSDownloadPublisher downloadToPublisher(final ClientSession clientSes
     public GridFSDownloadPublisher downloadToPublisher(final ClientSession clientSession,
                                                        final String filename,
                                                        final GridFSDownloadOptions options) {
-        return createGridFSDownloadPublisher(chunksCollection, notNull("clientSession", clientSession),
-                                             createGridFSFindPublisher(filesCollection, clientSession, filename, options));
+        Function<Timeout, GridFSFindPublisher> findPublisherCreator =
+                operationTimeout -> createGridFSFindPublisher(filesCollection, clientSession, filename,
```

Comment:
Nit: Keep the param names the same. Its `timeout`above and `operationTimeout` here.

Reply:
Renamed.